includes/ting_search_autocomplete.admin.inc

```diff
@@ -0,0 +1,210 @@
+      array(t('Facet index:', array(), array('context' => 'ting_search_autocomplete')),
+            t('The index to get suggestions from.<br/>
+               This variant looks in the specified index for entries that contain the words from specified query string and then counts results in groups based on the specified facet index. This will also provide suggestions for phrases that contain the words from the query string.<br/>
+               Care should be taken in selecting a suitable pair of indexes. The facet index should be an index generated from the same data as the match index. The match index should be a term index and the facet index should be a phrase index<br/>
+               F.ex.: scanphrase.mainTitle (combined with match index: scanterm.mainTitle) + query: "kan" will match "From Impressionism to <b>Kan</b>dinsky"', array(), array('context' => 'ting_search_autocomplete'))),
+      array(t('Query filter:', array(), array('context' => 'ting_search_autocomplete')),
+            t('A query that may be used to filter the area you want to get suggestions from. F.ex.: term.workType=literature', array(), array('context' => 'ting_search_autocomplete'))),
+      array(t('Output sort order:', array(), array('context' => 'ting_search_autocomplete')), t('Defaults to "count"', array(), array('context' => 'ting_search_autocomplete'))),
+      array(t('Agency:', array(), array('context' => 'ting_search_autocomplete')),
+            t('Only get suggestions from data accessible to the specified library ID. F.ex.: 715100', array(), array('context' => 'ting_search_autocomplete'))),
+      array(t('Profile:', array(), array('context' => 'ting_search_autocomplete')),
+            t('Only get suggestions from data accessible to the specified search profile, as defined in the VIP database.', array(), array('context' => 'ting_search_autocomplete'))),
+      array(t('Max suggestions:', array(), array('context' => 'ting_search_autocomplete')),
+            t('Maximum suggestions to return. Default 10', array(), array('context' => 'ting_search_autocomplete'))),
+      array(t('Max request time:', array(), array('context' => 'ting_search_autocomplete')),
+            t('Maximum request time in milliseconds. Default 2000', array(), array('context' => 'ting_search_autocomplete'))),
+      array(t('Highlight:', array(), array('context' => 'ting_search_autocomplete')),
+            t('Add prefix and suffix around strings that match the query.', array(), array('context' => 'ting_search_autocomplete'))),
+      array(t('Highlight prefix & suffix:', array(), array('context' => 'ting_search_autocomplete')), 
+            t('Defaults to &lt;em&gt; & &lt;/em&gt;.', array(), array('context' => 'ting_search_autocomplete'))),
+    ),
+    '#attributes' => array('class' => array('table-class')),
+    '#empty' =>t('Your table is empty'),
```

Comment:
I appreciate the help but why is this nested down here instead of in `#description` for each corresponding field?